### ISSUE TITLE
Costs for data with geo and without geo

### ISSUE THREAD

**Issue by virithavanama** [google/lightweight_mmm#68]
Hi Team, 
Is the cost sum of media_data for each channel with and without geo?
I have observed that in the demo with geo with 2 geo locations the cost should be doubled as media data is duplicated for 2 geo locations, but I see the cost remains same as without geo

**Comment by pabloduque0** [comment]
The costs refers to the sum of the media (assuming media is spend) per channel and geo. 

# Note that this is assuming media unit is spend (otherwise need to take price into account here).
costs = jnp.sum(media, axis=0) # for national data
costs = jnp.sum(media, axis=(0, -1)) # for geo data

For the mock data we simplify things a bit so some of the ROI numbers in the demo make sense.

**Comment by virithavanama** [comment]
Thank you
